packages/frontend/src/components/CreateSwap/CreateSwap.tsx

```diff
@@ -172,11 +180,13 @@ const CreateSwap = () => {
   };
 
   useEffect(() => {
-    if (tokens.length > 1) {
-      setValue(fromTokenKey, tokens[0].symbol);
-      setValue(toTokenKey, tokens[1].symbol);
+    if (fromTokens.length > 1) {
+      setValue(fromTokenKey, fromTokens[0].symbol);
+    }
+    if (toTokens.length > 1) {
+      setValue(toTokenKey, toTokens[1].symbol);
     }
-  }, [tokens]);
+  }, [fromTokens, toTokens]);
```

Comment:
Can we split this up into two useEffects? Right now, switching the `toChain` will reset the `toToken` and the `fromToken` which might not be the best UX.

Reply:
Addressed in 5a56209aab516cc6437c79e30a24605958be5ea1. Created a separate hook for this called `useDefaultTokens`.